docs/design/TECHNICAL_DOCUMENTATION.md

```diff
@@ -0,0 +1,142 @@
+# Design tools technical documentation
+
+Otter framework exposes a set of tools to optimize the interactions between the designers and the developers.
+The purpose of these tools is to reduce the steps between the design of the solution and the integration to the applications.
+
+## Design Token
+
+The [Design Token specification](https://design-tokens.github.io/community-group/format/) define a dialog format between the design tools (such as [Figma](https://www.figma.com)) and a style language (CSS , Sass, Less, etc...).
+The Otter Framework provides **Builder**, **Schematic** and **Command Line Interface** to generate CSS Variable and/or Sass Variable, based on the inputted Design Token files.
+
+### Global architecture of the Parser and Renderer
+
+The Design Token code generator tool is technically split in two main features :
+
+- The **parsers** that will parse and decode the Design Token Specification.
+- The **renderers** that will generate the output (mainly code) based on the decoded Design Token Specification.
+
+Both of these features can be customized at different levels. They can be customized thanks to parameters exposed by parsers and renderers **factory functions**, or they can be completely re-implemented and injected into the Design Token Code generator.
+
+### Parsers
+
+Currently the [o3r/design](https://github.com/AmadeusITGroup/otter/tree/main/packages/@o3r/design) package provides only a single [parser](https://github.com/AmadeusITGroup/otter/tree/main/packages/@o3r/design/src/core/design-token/parsers).
+This parser handle the Design Token Specification as JSON file (or loaded object resulting of a parsing of the JSON file) and result to an object facilitating the tree navigation and the final value calculation to the renderers.
+
+### Renderers
+
+The package is exposing 3 renderers:
+
+- **CSS** Renderer generating CSS Variables according to the specification *(default renderer)*
+- **Sass** Renderer generating Sass Variables according to the specification
+- **Metadata** renderer that will create the metadata file compatible with the Otter CMS solution.
```

Comment:
```suggestion
- **Metadata** The renderer that will create the metadata file compatible with the Otter CMS solution.
```
or 
```suggestion
- **Metadata**: Renderer that will create the metadata file compatible with the Otter CMS solution.
```

It could be nice to link a documentation - do we have one for the CMS solution?

Reply:
Maybe I would drop the "Renderer that will create" which seems a bit heavy word-wise.
I would prefer something like 
```suggestion
- **Metadata** can generate the metadata file compatible with the Otter CMS solution.
```